File: src/main/java/com/dy/dyweb/bean/Works.java
package com.dy.dyweb.bean;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Works {

    private long worksId;

    private long userId;

    private String title;

    private String url;

    private String username;

    private String userPic;

    private LocalDateTime createTime;

    private LocalDateTime updateTime;
}

File: data/src/main/java/com/aking/data/model/Works.kt
package com.aking.data.model
data class Works(
    val worksId: Long,
    val userId: Long,
    val title: String,
    val url: String,
    val username: String,
    val userPic: String,
    val createTime: String,
    val updateTime: String,
    val page: Int,
    val rows: Int
)
